public/app/plugins/panel/flamegraph/module.tsx

```diff
@@ -1,6 +1,12 @@
-import { PanelPlugin } from '@grafana/data';
+import { FieldConfigProperty, PanelPlugin } from '@grafana/data';
 
 import { FlameGraphPanel } from './FlameGraphPanel';
 import { FlameGraphSuggestionsSupplier } from './suggestions';
 
-export const plugin = new PanelPlugin(FlameGraphPanel).setSuggestionsSupplier(new FlameGraphSuggestionsSupplier());
+const flamegraphConfigOptions = [FieldConfigProperty.Unit, FieldConfigProperty.Decimals]
```

Comment:
```suggestion
const flamegraphConfigOptions = [FieldConfigProperty.Unit, FieldConfigProperty.Decimals];
```

I think colon is missing at the end and that's why build is failing at linting